src/main/java/mil/dds/anet/database/AttachmentDao.java

```diff
@@ -0,0 +1,174 @@
+package mil.dds.anet.database;
+
+import io.leangen.graphql.annotations.GraphQLRootContext;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+import mil.dds.anet.AnetObjectEngine;
+import mil.dds.anet.beans.Attachment;
+import mil.dds.anet.beans.AttachmentRelatedObject;
+import mil.dds.anet.beans.search.AbstractSearchQuery;
+import mil.dds.anet.database.mappers.AttachmentMapper;
+import mil.dds.anet.database.mappers.AttachmentRelatedObjectMapper;
+import mil.dds.anet.utils.DaoUtils;
+import mil.dds.anet.utils.FkDataLoaderKey;
+import mil.dds.anet.views.ForeignKeyFetcher;
+import ru.vyarus.guicey.jdbi3.tx.InTransaction;
+
+public class AttachmentDao extends AnetBaseDao<Attachment, AbstractSearchQuery<?>> {
+
+  public static final String TABLE_NAME = "attachments";
+
+  @Override
+  public Attachment getByUuid(String uuid) {
+    return getByIds(Arrays.asList(uuid)).get(0);
+  }
+
+  static class SelfIdBatcher extends IdBatcher<Attachment> {
+    private static final String sql = "/* batch.getAttachmentByUuids */ "
+        + "SELECT * FROM \"attachments\" WHERE uuid IN ( <uuids> )";
```

Comment:
Explicitly specify the column, and don't select the contents.
The contents should only be retrieved from the database if the user views or downloads the attachment (and that should be a separate database query), in all other cases only the metadata columns should be retrieved.